### PR TITLE
wlip-0002 Remove broken links in Examples section

### PR DESCRIPTION
**Pull request by jasikor**
I suggest to remove broken links in Examples sections for the following reasons:

1. Broken links make an impression that wlip-0002 is at the very early stage of development. 
2. I bet BIP39 authors would be more keen to merge WLIPS if there were no broken links.
3. Example section is intended to show some file names, not files themselves. Links in this context are unnecessary.
4. Linking to the file with broken hash (namely `English-770d1896-obsolete`) might motivate somebody to indeed check the hash and then bother @p2w34 with unnecessary requests :)

### CODE REVIEW

**Comment by p2w34** [comment]
Thanks for spotting invalid hash, although merging in this commit would be only a half-measure, as it leaves the file itself with incorrect hash. Id rather recompute the hash. The links may stay or may be removed - I am fine with both approaches.

**Comment by jasikor** [comment]
I will recompute the hash tomorrow morning😀

File: wlip-0003.md [hunk 1]
```diff
@@ -26,6 +26,6 @@ as English-770d1896-obsolete.
 |Word list|Character set|Status|Comment|
 |---|---|---|---|
 |[English-missing_hash](wlip-0003/english-missing_hash)|English|Draft|
-|[English-770d1896-obsolete](wlip-0003/english-770d1896-obsolete)|English|Draft|considered obsolete as does not comply with WLIP-0003|
+|[English-a1d03317-obsolete](wlip-0003/english-a1d03317-obsolete)|English|Draft|considered obsolete as does not comply with WLIP-0001|
```
Comment (p2w34) [inline wlip-0003.md:29]: There is one more mention of english-770d1896-obsolete is this file

**Comment by p2w34** [comment]
Looks good to me.

**Pull request merged**